client/src/components/Workflow/Editor/Index.vue

```diff
@@ -878,7 +897,7 @@ export default {
         async saveOrCreate() {
             if (this.hasInvalidConnections) {
```

Comment:
```suggestion
            if (this.hasInvalidConnections ||this.hasDuplicateOutputs) {
```

Do we want to display the confirmation also in this case?

Reply:
good catch!